integration_tests/CMakeLists.txt

```
@@ -107,6 +107,29 @@ macro(RUN)
             if (${RUN_FAIL})
                set_tests_properties(${name} PROPERTIES WILL_FAIL TRUE)
             endif()
+        elseif(KIND STREQUAL "wasm")
+            # wasm test
+            execute_process(COMMAND lpython --backend wasm ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py -o ${name})
```

Please, could someone possibly share if it is fine to use `execute_process` in `integration_tests/CMakeLists.txt` to test the `wasm` `integration_tests`?

---

We can later investigate if this would work:
```suggestion
             add_custom_command(
                OUTPUT ${name}.o
                COMMAND lpython --backend wasm ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py -o ${name}.o
                DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py
                VERBATIM)
```